code/datums/ammo/bullet/bullet.dm

```diff
@@ -0,0 +1,81 @@
+/*
+//======
+					Default Ammo
+//======
+*/
+//Only when things screw up do we use this as a placeholder.
+/datum/ammo/bullet
+	name = "default bullet"
+	icon_state = "bullet"
+	headshot_state = HEADSHOT_OVERLAY_LIGHT
+	flags_ammo_behavior = AMMO_BALLISTIC
+	sound_hit  = "ballistic_hit"
+	sound_armor  = "ballistic_armor"
+	sound_miss  = "ballistic_miss"
+	sound_bounce = "ballistic_bounce"
+	sound_shield_hit = "ballistic_shield_hit"
+
+	accurate_range_min = 0
+	damage = 10
+	shrapnel_chance = SHRAPNEL_CHANCE_TIER_1
+	shrapnel_type = /obj/item/shard/shrapnel
+	shell_speed = AMMO_SPEED_TIER_4
+
+/datum/ammo/bullet/proc/handle_battlefield_execution(datum/ammo/firing_ammo, mob/living/hit_mob, obj/projectile/firing_projectile, mob/living/user, obj/item/weapon/gun/fired_from)
```

Comment:
I appreciate that this is a refactor from code without any changes - it feels weird for all bullet code to handle BEs. Maybe this would be a good candidate to refactor into a datum (future change)?

Reply:
i can make proc that handle battlefield execution it's own file?
it probably shouldn't be link to bullet...
could also make it be in datum/ammo instead....